src/remotepublish.cmd.js

```diff
@@ -155,6 +158,30 @@ class RemotePublishCommand extends AbstractCommand {
     return this;
   }
 
+  withCustomVCL(value) {
```

Comment:
no jsdoc?

Reply:
ah ah. I adopted the jsdoc pattern of that file :) but you are right, that's not a reason, we need to start somewhere.